src/forecast.py

```diff
@@ -213,8 +190,72 @@ def forecast_page():
             )
         )
 
-    st.plotly_chart(fig, theme="streamlit")
-
+    # Fetch and plot Elexon Solar Forecast data
+    st.sidebar.subheader("Select Elexon Forecast Dates")
+    start_datetime_utc = st.sidebar.date_input("Start Date", datetime.utcnow() - timedelta(days=3))
```

Comment:
could you use the same start and end times as above, then we dont need these extra options